### ISSUE TITLE
 Attempt to convert a value (1.0) with an unsupported type (<class 'numpy.float32'>) to a Tensor.

### ISSUE THREAD

**Issue by Vidyapreethaam** [tensorflow/tensorflow#33154]
Im getting this error in this step. Im trying to forecast sales similar to the lstm stock price prediction. I ensured that all the inputs i.e x_t_tf, y_t_tf, a and b are all tensors. Yet, I get this error. I am using tensorflow 2.0.0.

history = model.fit(x_t_tf, y_t_tf, epochs=300, verbose=2, steps_per_epoch=BATCH_SIZE,validation_steps = 3,
                        shuffle=False, validation_data=(a,
                        b), callbacks=[es, mcp])

Please let me know how to resolve this. Im just using a jupyter notebook to run this.

**Comment by gowthamkpr** [comment]
@Vidyapreethaam Can you provide a gist of this issue (preferably a colab notebook) inorder for me to reproduce this.  Its too vague for me to look into this issue now. Thanks!

**Comment by Vidyapreethaam** [comment]
Im following steps similar to this article 
https://towardsdatascience.com/predicting-stock-price-with-lstm-13af86a74944 
similar to this code
https://github.com/DarkKnight1991/Stock-Price-Prediction/blob/master/stock_pred_main.py

and in this part, the history= .fit() is creating the issue. So, in my code I ensured that all the inputs are tensors. yet Im getting this error. 

Please look at the code above.

**Comment by Vidyapreethaam** [comment]
Also, Im not using colab for this. Im doing it in a jupyter notebook

**Comment by Vidyapreethaam** [comment]
[stockprice-master.zip](https://github.com/tensorflow/tensorflow/files/3704566/stockprice-master.zip)
please also check this notebook. If we run the .fit() step. It gives the same error.

**Comment by gowthamkpr** [comment]
@Vidyapreethaam Please post this issue in [stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow) as github is only meant for issues related to bug, performance, feature requests, build/install and docs. Thanks!
